contrib/aur/bitcoin-abc-qt/PKGBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=bitcoin-abc-qt
-pkgver=0.23.9
+pkgver=0.24.0
```

Comment:
This shouldn't be here. Probably left in the previous merge by accident. Need to delete before merging.